src/common/sorted-table/index.js

```diff
@@ -569,21 +594,14 @@ export default class SortedTable extends Component {
     const { target } = event
     this._selectItem(+target.name, target.checked, event.nativeEvent.shiftKey)
   }
-
-  _executeGroupedAction = handler => {
-    const { state } = this
-    return handler(
-      state.all
+  _getSelectedItems = createSelector(
+    () => this.state.all,
+    () => this.state.selectedItemsIds,
+    (all, selectedItemsIds) =>
+      all
         ? this._getItems()
-        : filter(this._getItems(), item => state.selectedItemsIds.has(item.id))
-    )
-  }
-
-  _executeRowAction = event => {
-    const { props } = this
-    const item = this._getVisibleItems()[event.currentTarget.dataset.index]
-    props.rowAction(item, props.userData)
-  }
+        : filter(this._getItems(), item => selectedItemsIds.has(item.id))
```

Comment:
@julien-f Isn't it weird to explicitly call a selector in the result function of another selector (instead of passing it as an input selector)? I don't see why it wouldn't work in this case but it feels unusual.

Reply:
It's usually a bad idea because it indicates a hidden dependency.